common/api/core-bentley.api.md

```diff
@@ -1211,6 +1211,9 @@ export class LRUMap<K, V> extends LRUCache<K, V> {
     constructor(limit: number);
 }
 
+// @public
+export type MarkRequired<T, K extends keyof T> = Pick<Required<T>, K> & Omit<T, K>;
```

Comment:
I hate naming things and this is a nitpick, but type names are generally nouns while "mark required" is an action. `RequiredSubset` better?

Reply:
I stole the name from the [ts-essentials package](https://github.com/krzkaczor/ts-essentials#markrequired), i prefer the precedent, thoughts?

Reply:
My preference is for nouns but like I said, it's a nitpick - feel free to disregard.

Reply:
Yeah I would be willing to change it if it weren't for that precedent name for that utility type elsewhere I've used it, I'll keep it in this case. Please resolve unless you see any issues.